server.go

```diff
@@ -208,6 +209,24 @@ func StatsHandler(h stats.Handler) ServerOption {
 	}
 }
 
+// UnknownServiceHandler returns a ServerOption that allows for adding a custom
+// unknown service handler. The provided method is a bidi-ServerStream handler
```

Comment:
Seems to me `bidi-ServerStream` is confusing.
How about just `bidi-streaming RPC service handler`?